src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/params.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import json
+from ._util import ParametersContext
+
+with ParametersContext(command='monitor alert-rules') as c:
+    c.register_alias('name', ('--azure-resource-name',))
+    c.register_alias('rule_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor alert-rule-incidents') as c:
+    c.register_alias('incident_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor log-profiles') as c:
+    c.register_alias('log_profile_name', ('--name', '-n'))
+
+with ParametersContext(command='monitor autoscale-settings') as c:
+    c.register_alias('name', ('--azure-resource-name',))
```

Comment:
`--azure-resource-name` is pretty verbose. Also, will this accept a name or id?  Should it?  If so, I would recommend `--resource` with help text that says "Name or ID of the resource to query." or something to that effect.